doc/links.rst

```diff
@@ -1,20 +1,25 @@
-
+.. _`Cookiecutter`: https://cookiecutter.readthedocs.io
+.. _`pluggy`: https://pluggy.readthedocs.io
+.. _`cookiecutter-tox-plugin`: https://github.com/tox-dev/cookiecutter-tox-plugin
 .. _devpi: http://doc.devpi.net
 .. _Python: http://www.python.org
-.. _virtualenv: https://pypi.python.org/pypi/virtualenv
-.. _virtualenv3: https://pypi.python.org/pypi/virtualenv3
-.. _virtualenv5: https://pypi.python.org/pypi/virtualenv5
-.. _`py.test`: http://pytest.org
+.. _virtualenv: https://pypi.org/project/virtualenv
+.. _virtualenv3: https://pypi.org/project/virtualenv3
+.. _virtualenv5: https://pypi.org/project/virtualenv5
```

Comment:
this returns page  not found - virtualenv5

Reply:
Linkchecks don't complain - looks like pypi.org does nont answer 404 if a project does not exist. I removed both versioned links, as they weren't used anymore anyway.